root/src/main/java/com/example/servingwebcontent/Movie.java

```diff
@@ -32,12 +37,20 @@ public String getFilmRating () {
 		return filmRating;
 	}
 
-	public Movie (String id, String type, String name, String releaseDate, int duration, String filmRating) {
-		this.id = id;
+	public static final String [] getColumns () {
```

Comment:
just fyi, pretty much everyone in industry do not include spaces between the data type and [] (i.e. String [] is written as String[]) and same thing for parentheses (i.e. getColumns () is usually written as getColumns())